packages/mc-scripts/src/utils/get-config-diff.js

```diff
@@ -0,0 +1,246 @@
+const { red, green } = require('chalk');
+
+const notNestedKeySet = new Set(['name', 'description', 'url', 'icon']);
+
+const t = (tabs) => '\t'.repeat(tabs);
+
+const getStringDiff = (oldStr, newStr, key, tabs = 0) => {
+  if (!oldStr && newStr) {
+    return `${t(tabs)}${key} added: ${green(newStr)}`;
+  } else if (oldStr && !newStr) {
+    return `${t(tabs)}${key} removed: ${red(oldStr)}`;
+  } else if (oldStr && newStr && oldStr !== newStr) {
+    return `${t(tabs)}${key} changed: ${red(oldStr)} => ${green(newStr)}`;
+  }
+};
+
+const getPermissionsDiff = (oldPermissions, newPermissions) => {
+  const permissionDiff = ['permissions changed'];
+  const mappedOldPermission = {};
+
+  oldPermissions.forEach(({ name, oAuthScopes }) => {
+    mappedOldPermission[name] = oAuthScopes;
+  });
+
+  newPermissions.forEach((newPermission) => {
+    const currDiff = [`\t${newPermission.name} changed`];
```

Comment:
I noticed that sometimes you use `t(tabs)` and for example here you use `\t` directly. Let's try to be a bit more consistent maybe.